utils/utils.go

```diff
@@ -130,11 +131,22 @@ func PrepareLocalPathForUpload(localPath string, useRegExp bool) string {
 		localPath = localPath[3:]
 	}
 	if !useRegExp {
-		localPath = pathToRegExp(localPath)
+		localPath = pathToRegExp(cleanPath(localPath))
 	}
 	return localPath
 }
 
+// Clean /../ | /./ and return the double backslash in case of windows path
```

Comment:
I suggest changing this comment to the following:
 // Clean /../ | /./ using filepath.Clean.

Reply:
Fixed